pkgs/development/tools/misc/texinfo/6.0.nix

```diff
@@ -9,8 +9,11 @@ stdenv.mkDerivation rec {
   };
 
   buildInputs = [ perl xz ]
+    ++ stdenv.lib.optional stdenv.isSunOS libiconv
     ++ stdenv.lib.optional interactive ncurses;
 
+  configureFlags = "AWK=${gawk}/bin/awk";
```

Comment:
Actually, I think that's exactly the purpose of this `common-path.nix` file. Darwin and Linux stdenvs do use it (the only two significantly active platforms these years).